cypress/component/ValueAnnotation.cy.tsx

```diff
@@ -18,4 +18,30 @@ describe('ValueAnnotation', () => {
       .should('be.visible')
       .and('contain', 'Please select the appropriate data type for this column.');
   });
+  it('asserts that there is no shared state between EditDescription components in Continuous component', () => {
+    useDataStore.setState({ columns: mockColumnsWithDataType });
+    useDataStore.setState((state) => ({
+      columns: {
+        ...state.columns,
+        2: {
+          ...state.columns[2],
+          dataType: 'Continuous',
+        },
+      },
+    }));
+
+    cy.mount(<ValueAnnotation />);
+    cy.get('[data-cy="side-column-nav-bar-continuous-participant_id"]').click();
+    cy.get('[data-cy="1-continuous"]').should('be.visible');
```

Comment:
these types of assertions aren't really related to the test case, right? i.e. it's just about "if I type over here, does it change over there"

Reply:
Yea they can be removed.